src/sentry/api/endpoints/organization_code_mapping_details.py

```diff
@@ -28,6 +28,23 @@ def convert_args(self, request, organization_slug, config_id, *args, **kwargs):
 
         return (args, kwargs)
 
+    def _get_codeowner_contents(self, config):
+        integration = config.organization_integration.integration
+        install = integration.get_installation(config.organization_integration.organization_id)
+        return install.get_codeowner_file(config.repository, ref=config.default_branch)
+
+    def get(self, request, config_id, organization, config):
```

Comment:
@MeredithAnya this endpoint looks like its returning the same info as GET OrganizationCodeMappingCodeOwnersEndpoint. Do we need this?

Reply:
oh yeah sorry this is old, lemme delete